profiles/sssd/password-auth

```diff
@@ -20,11 +21,13 @@ account     sufficient                                   pam_localuser.so
 account     sufficient                                   pam_usertype.so issystem
 account     [default=bad success=ok user_unknown=ignore] pam_sss.so
 account     required                                     pam_permit.so
+-account    sufficient                                   pam_systemd_home.so                                    {include if "with-homed"}
 
 password    requisite                                    pam_pwquality.so local_users_only
 password    sufficient                                   pam_unix.so yescrypt shadow {if not "without-nullok":nullok} use_authtok
 password    sufficient                                   pam_sss.so use_authtok
 password    required                                     pam_deny.so
+-password   sufficient                                   pam_systemd_home.so                                    {include if "with-homed"}
```

Comment:
here too, this makes no sense to put after pam_deny, as the request will then be refused already. this should be put first in this stack actually, since we don't want to run the rest of the stack at all for homed users (it does its own pwquality anyway, no need to repeat it)

Reply:
Done